### PR TITLE
Implemented interfaces to read the configuration for stakes and genesis block

### PR DESCRIPTION
**Pull request by easye**
Use EMOTIQ/CONFIG:GET-GENESIS-BLOCK to get the configured genesis block.

### CODE REVIEW

File: src/stakes.lisp [hunk 1]
```diff
@@ -18,3 +18,13 @@
     (format o ";;; THIS FILE IS FOR TESTING ONLY~%")
     (dolist (stake records)
       (format o "~s~%" stake))))
+
+(defun get-stakes ()
+  (with-open-file (o (make-pathname :name "stakes" :type "conf"
+                                    :defaults (emotiq/fs:etc/))
+                     :direction :input)
+    (loop :with form
+       :while (setf form (read o nil nil nil)) :collecting form)))
```
Comment (markhdavid) [inline src/stakes.lisp]: You must bind variable ```*READ-SUPPRESS*``` true around a call to read on random foreign input.

**Review by markhdavid — CHANGES_REQUESTED** [review]
Needs ```*READ-SUPPRESS*```, see comment

File: src/stakes.lisp (hunk 1 above)
Comment (easye) [reply src/stakes.lisp]: > bind variable *READ-SUPPRESS*

Done in 0bb7bc2

**Review by markhdavid — APPROVED** [review]
looks good

**Pull request merged**